### DISCUSSION TITLE
Abstract Usage, To/From

### DISCUSSION THREAD

**Discussion by rodney528** [Kriptel/RuleScript#17] [Q&A] [answered]
So after reading the code of RuleScript it looked like hscript can mess with Abstracts made in source. After some testing though I wanted to ask how well RuleScript can handle To/From shenanigans.

**Comment by Kriptel** [comment, marked as answer]
For now, AbstractMacro can help a script use static abstract fields. I'm going to implement full abstract functionality in future versions

**Reply by rodney528** [reply]
Awesome! Because uhhhhhhhh, my entire pathing system kinda revolves around a single abstract. 🥲
